src/mono/mono/component/debugger-agent.c

```diff
@@ -7089,7 +7126,7 @@ event_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 				if (exc_class) {
 					req->modifiers [i].data.exc_class = exc_class;
 
-					if (!mono_class_is_assignable_from_internal (mono_defaults.exception_class, exc_class)) {
+					if (!mono_class_is_assignable_from_internal (get_mono_defaults ()->exception_class, exc_class)) {
```

Comment:
So for this I would do the fastpath trick I mentioned earlier.   Make a variable in the debugger component like `mdbg_mono_defaults` and add a new method that sets up the debugger component (or reuse one of the other methods that get called early) and pass `mono_defaults` from the runtime and assign it to `mdgb_mono_defaults`.  and then here you can use `mdbg_mono_defaults.exception_class`.

Reply:
done.